index.bs

```diff
@@ -1097,6 +1098,13 @@ The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "S
 :: A [=credential property=] is some characteristic property of a [=public key credential source=], such as whether it is a
     [=client-side discoverable credential=] or a [=server-side credential=].
 
+: <dfn>Hardware-bound Device Key Pair</dfn>
+: <dfn>Device-bound Key</dfn>
+: <dfn>Device Private Key</dfn>
+: <dfn>Device Public Key</dfn>
+:: A [=hardware-bound device key pair=], also known as a [=device-bound key=], is an [=authenticator=]-, [=[RP]=]-, and [=user credential=]-specific public key pair created upon a [=[RP]=]'s request via the [=devicePubKey=] [=WebAuthn extension=].
```

Comment:
Isn't the dpk always created if the authenticator supports DPK and the public key credential source is multi-device?

eg even if the RP doesn't request DPK in the make credential it would still be available if the RP sends the extension in getAssertion along with the DPK attestation if there is one..

Reply:
No. If the RP does not request a DPK, an authenticator does not have to generate one.

Reply:
if I understand correctly, both of the above comments are saying the same thing.

also where @ve7jtb wrote "DPK attestation", I think he meant "DPK extension", yes?

Reply:
The attestation returned in the DPK extension.

Reply:
ok, so what I think @ve7jtb meant to write is:
> eg even if the RP doesn't request DPK in a makeCredential() call, it would still be returned, along with DPK attestation (if any), if the RP subsequently sends the DPK extension in getAssertion() 

...?

Reply:
Again it sounds to me like you are both saying the same thing. 

I suggest we clarify this detail down in the [[#sctn-device-publickey-extension]] section rather than trying to do so in the [=device-bound key pair=] (was: [=hardware-bound device key pair=]) definition.